.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## Background
+<!-- What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary -->
+
+## Definition of Done
```

Comment:
I like this distinction between AC and DoD. In the past, I think we've used them interchangeably (which has caused me some confusion), so I'm wondering if we should accompany this with a definition for each term in some documentation somewhere.